websites/A/Aniorb/presence.ts

```diff
@@ -0,0 +1,107 @@
+const presence = new Presence({
+  clientId: "776312991939428373"
+}),
+strings = presence.getStrings({
+  play: "presence.playback.playing",
+  pause: "presence.playback.paused",
+  browsing: "presence.activity.browsing"
+});
+let video = {
+video: false,
+duration: 0,
+currentTime: 0,
+paused: true
+};
+
+/**

```

Comment:
Remove the get Timestamps function entirely and its comments and replace instances with `presence.getTimestamps`